### ISSUE TITLE
Prevent autocomplete to change position (set fix position)

### ISSUE THREAD

**Issue by Psyphant** [webscopeio/react-textarea-autocomplete#152]
Is there a way to prevent the autocomplete opening at the top of the textarea?

Because on the one hand the change is annoying and on the other it's in my case buggy:
https://s3.gifyu.com/images/overflow-bug.gif

**Comment by jukben** [comment]
Have you tried to play with `boundariesElement` prop? Maybe it would be good to allow `false` option to disable it.

**Comment by Psyphant** [comment]
Yeah, but unfortunately `boundariesElement` does not work with (because the autocomplete have to be inside the boundariesElement) or without `renderToBody`.

So ... an additional prop will be great!

For example like:
`autocompletePostionX`: auto | left | right
`autocompletePostionY`: auto | top | bottom

**Comment by jukben** [comment]
Yeah, agree, more like

```jsx
autocompletePostion={{x: "auto | left | right", y: "auto | left | right"}}
```
defaults to `auto`

**Comment by Psyphant** [comment]
> Yeah, agree, more like
> 
> ```js-jsx
> autocompletePostion={{
> 	x: "auto | left | right",
> 	y: "auto | top | bottom"
> }}
> ```
> 
> defaults to `auto`
> 
> What do you think?

Sounds sweet to me 😎

**Comment by jukben** [comment]
I can try tackle it on weekend, but if you have some time for it, you have green light!

**Comment by Psyphant** [comment]
Sounds great!

I had another case in which the autocomplete has the wrong position (renderToBody is true):
![image](https://user-images.githubusercontent.com/364954/57363096-94efb200-7180-11e9-971c-c1f50f635445.png)

Maybe the calculation of the autocomplete position with the new renderToBody prop is not correct.

**Comment by Psyphant** [comment]
Hey!

I invest some time finding the root cause of the problem:

When the autocomplete has a bottom position even there is no space for it, the variable `var dropdownBounds = this.ref.getBoundingClientRect();` (line 601) is empty. 

So the problem might be the async loading of the dropdown.

The solution could be to set the position after the dropdown finished loading.

**Comment by Psyphant** [comment]
The wrong calculation of the autcomplete position was my bad 🤦‍♂ `.rta__autocomplete` was set to `display: none;` by default ... so the `dropdownBounds` variable was empty. Now it works. 

Anyway the `autocompletePostion` prop would be great 👍
